workshop-1462877774/french.po

```diff
@@ -64319,7 +64319,7 @@ msgstr "Bah ! Il faut plus de temps."
 #. STRINGS.CHARACTERS.WEBBER.ACTIONFAIL.ROW_FAIL.BAD_TIMING2
 msgctxt "STRINGS.CHARACTERS.WEBBER.ACTIONFAIL.ROW_FAIL.BAD_TIMING2"
 msgid "Row row row your boat... maybe a bit better next time."
-msgstr "Row Row Rangez votre bateau... peut-être un peu mieux la prochaine fois."
+msgstr "Row Row Row your boat... peut-être un peu mieux la prochaine fois."
```

Comment:
Note: il s'agit d'une référence à une chanson classique ("row row row your boat, gently down the stream...")